hack/test

```diff
@@ -61,6 +62,14 @@ if [ "$TEST_COVERAGE" = "1" ]; then
   coverageFlags="-coverprofile=/coverage/coverage.txt -covermode=atomic"
 fi
 
+if [ -n "$TEST_WORKER" ]; then
+  if [[ $TESTFLAGS == *"--run"* ]]; then
```

Comment:
`-test.run` `-run` etc are also valid here

Reply:
I moved the logic into the workflow to make it less cumbersome in the hack script. In a follow-up I think we could "bakerized" to streamline the workflow.